tests/components/vicare/conftest.py

```diff
@@ -35,10 +38,28 @@ def __init__(self, fixtures: list[str]) -> None:
 class MockViCareService:
     """PyVicareService mock using a json dump."""
 
-    def __init__(self, fixture: str) -> None:
+    def __init__(
+        self,
+        installation_id: str,
+        gateway_id: str,
+        device_id: str,
+        fixture: tuple[list[str], str],
```

Comment:
Can we use a namedtuple or maybe a dataclass for it?
It would improve readability